learn/wavemaker-release-notes/v10-5-6.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "WaveMaker 10.5.6 - Release date: 08 October 2020"
+id: ""
+sidebar_label: "Release v10.5.6"
+---
+*WaveMaker released a new update version with bug fixes.*
+
+## Improvements
+
+---
+
+- Stackable multiple toaster messages. This offers a way to display multiple toasters where you can configure the time duration.
+- Update of Android **targetSdkVersion** is required by November 2, 2020. For more information, see [Mobile Build - Manual](learn/hybrid-mobile/mobile-build-manual).
+
+## Bug Fixes
+
+---
+
+- Fixed issue related to exporting project shell to EDN.
+- Fixed issues related to editing and displaying the user profile name.
+- Fixed issue related to default messages which are not localized.
+- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
+- Fixed issue related to the `reset()` method that was not working for an HTML upload widget.
+- Fixed issue related to accessing the second level Partial page widgets from `App.activePage`.
+- Fixed issue related to the **Animation Interval** property for the **Carousel** widget to set it through the script.
+- Fixed issue related to the **Chips** widget dataset that is not updating the UI when the variable is updated.
+
+### Data Table
+
+- Fixed issue related to DataTable column styles that are given in %.
+- Fixed issue related to the **INFO** toaster for Inline Edit in the DataTable row.
+- Fixed issues related to applying stypes for Data Table, including "table-bordered" and "table-striped" not migrating from 9.x to 10.x version.
+- Fixed issue related to the pagination based on the **Records per Request** property in the Data Table.
+
```

Comment:
More bug fixes that can be added:
* Fixed scrolling issues with content inside a design dialog.
* Fixed few broken documentation links in the studio.